recipe/meta.yaml

```
@@ -33,6 +33,9 @@ source:
   - url: https://static.rust-lang.org/dist/rust-std-{{ version }}-x86_64-apple-ios.tar.gz  # [(linux or win) and x86_64]
     sha256: 95f8300e5421141f9f9a8f45b866106724e00d9f307d7b880a046cddce5f73d4  # [(linux or win) and x86_64]
     folder: rust-std  # [(linux or win) and x86_64]
+  - url: https://static.rust-lang.org/dist/rust-std-{{ version }}-x86_64-pc-windows-gnu.tar.gz  # [(linux or win) and x86_64]
```

Can you try this with MSVC?
```suggestion
  - url: https://static.rust-lang.org/dist/rust-std-{{ version }}-x86_64-pc-windows-msvc.tar.gz  # [(linux or win) and x86_64]
```

---

My impression was that the whole point was to target gnu, not msvc?

---

Sorry. The problem here is missing context as conversation was happening also in private channels. This should be the base to cross-compile polars from Linux to Windows. To make it compatible with the rest of Rust on conda-forge, we should also use MSVC in the cross-compiling case.